config/testgrids/openshift/redhat-openshift-ocp-release-4.4-informing.yaml

```diff
@@ -1433,9 +1406,6 @@ test_groups:
 - days_of_results: 60
   gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-ovn-4.4
   name: release-openshift-ocp-installer-e2e-aws-ovn-4.4
-- days_of_results: 60
-  gcs_prefix: origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-proxy-4.4
-  name: release-openshift-ocp-installer-e2e-aws-proxy-4.4
```

Comment:
we're losing the proxy job from informing?

Reply:
Job was deleted by someone in openshift/release@9b767976ce295e875c20766a8bd60ba8537c033e

Reply:
You should track with https://github.com/openshift/release/commit/9b767976ce295e875c20766a8bd60ba8537c033e#r40765657